pkg/tests/end_to_end_tests/ha_multiple_promscales_test.go

```diff
@@ -0,0 +1,230 @@
+package end_to_end_tests
+
+import (
+	"context"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"testing"
+	"time"
+)
+
+// setLeaseExplicitly tries to set the ha lease to
+// what is sent in the state argument. If nil state
+// sent, no action taken.
+// returns:
+//    - an error if it couldn't set the state
+func setLeaseExplicitly(db *pgxpool.Pool, state *leaseState) error {
+	if state == nil {
+		return nil
+	}
+	sql := `UPDATE ha_leases SET leader_name=$1, lease_start=$2, lease_until=$3
+			WHERE cluster_name=$4`
+	_, err := db.Exec(context.Background(), sql, state.leader, state.leaseStart, state.leaseUntil, state.cluster)
```

Comment:
(optional) Should we add an error trace here, than simply sending back? Maybe helpful then some test goes wrong.
like `fmt.Errorf("setLeaseExplicitly: %w", err)`

Reply:
only called from 1 place in the tests, and if != nil fails (and that logs it) immediately